src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createNotification = (text, cssClass = '') =>
+  `<div class="message ${cssClass}">${text}</div>`;
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
```

Comment:
Recommendation: It's a good practice to simulate some delay before resolving a Promise to mimic real-life situations. You can use `setTimeout` function for this.